src/main/java/com/risevision/bigquery/BigqueryCommonApi.java

```diff
@@ -0,0 +1,36 @@
+package com.risevision.bigquery;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.bigquery.Bigquery;
+import com.google.api.services.bigquery.BigqueryScopes;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public class BigqueryCommonApi {
```

Comment:
@tejohnso do you have any specific alternatives in mind?

I don't remember how I arrived at this solution, but I toyed around with a few ideas before separating out the common functionality.